src/main/java/io/cryostat/agent/TruststoreConfig.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright The Cryostat Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.cryostat.agent;
+
+public class TruststoreConfig {
+    private String alias;
+    private String path;
+    private String type;
+
+    public TruststoreConfig(String alias, String path, String type) {
```

Comment:
It's not super important, but I'd rather this class be immutable (ie all fields `final`). Given how they are constructed, using the config property keys and partially building each one as the key/value pairs are encountered, I think this could be accomplished using a Builder pattern?

Reply:
And relating to the other thread about the null config values, should this constructor assert that the fields passed in are non-null?